src/components/Button/Button.tsx

```diff
@@ -25,12 +26,14 @@ export const Button: React.FC<ButtonProps> = ({
   iconRight,
   ...restProps
 }: ButtonProps) => {
+  const isSmallWindow = useIsUnderWindowBreakpoint({ size: "sm" });
```

Comment:
First thing: We don't need this unless it's configurable with a prop, it can all be done with CSS which is way more performant

Second thing: I think this does not belong in the Button, we can drive the whole thing through ConnectWallet. It's currently changing the behavior of all buttons which is not what we want